### DISCUSSION TITLE
binary installers?

### DISCUSSION THREAD

**Discussion by Phillip-M-Feldman** [BurntSushi/ripgrep#1646] [General] [answered]
I'm looking for binary installers for RipGrep 11 and RipGrep 12, for Windows 10 and RedHat Linux, but haven't been able to find these.

**Comment by BurntSushi** [comment, marked as answer]
There is no installer. As per the [installation instructions](https://github.com/BurntSushi/ripgrep#installation), you just download the zip archive from the [releases](https://github.com/BurntSushi/ripgrep/releases), unzip it and it contains the `exe` file.

**Comment by Phillip-M-Feldman** [comment]
Thanks so much, and my apologies for not reading more carefully!
